modules/admission-webhook-about.adoc

```diff
@@ -0,0 +1,52 @@
+
+* If any of the webhooks deny the request, the admission request is denied and the reason for doing so is based on the first denial.
+
+* If more than one webhook denies the admission request, only the first is returned to the user.
+
+* If there is an error encountered when calling a webhook, that request is either denied or the webhook is ignored.
+
+//Future xrefs - The communication between the admission controller and the webhook server must use TLS. Generate a certificate authority (CA) certificate and use the certificate to sign the server certificate that is used by your webhook server. The PEM-encoded CA certificate is supplied to the admission controller using a mechanism, such as xref:../authentication/certificates/service-serving-certificate.adoc#service-serving-certificate[service serving certificate secrets].
+The communication between the admission controller and the webhook server must use TLS. Generate a CA certificate and use the certificate to sign the server certificate that is used by your webhook server. The PEM-encoded CA certificate is supplied to the admission controller using a mechanism, such as service serving certificate secrets.
+
+The following diagram illustrates the sequential admission chain process within which multiple webhook servers are called.
+
+.API admission chain with mutating and validating webhooks
+image::api-admission-chain.png["API admission stage", align="center"]
+
+An example use case for admission webhooks is syntactical validation of resources. For example, in an infrastructure where all pods must have a common set of labels, a webhook can inject labels and another webhook can verify that labels are as expected. {product-title} would subsequently schedule pods that include required labels and reject those that do not.
+
+Some common admission webhook use cases include:
+
+//Future xref - * Namespace reservation.
+* Namespace reservation.
+//Future xrefs - * xref:../networking/hardware_networks/configuring-sriov-operator.adoc#configuring-sriov-operator[Limiting custom network resources managed by the SR-IOV network device plug-in].
+* Limiting custom network resources managed by the SR-IOV network device plug-in.
+//Future xref - * xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations_dedicating_nodes-scheduler-taints-tolerations[Defining tolerations that enable taints to qualify which pods should be scheduled on a node].
+* Defining tolerations that enable taints to qualify which pods should be scheduled on a node.
+//Future xref - * xref:../nodes/pods/nodes-pods-priority.adoc#admin-guide-priority-preemption-names_nodes-pods-priority[Pod priority class validation].
+* Pod priority class validation.
+
+[NOTE]
+====
+The `projectrequestlimits` admission webhook, which enables administrators to set a limit to how many namespaces can be created per user account, is not currently supported within {product-title} {product-version}.
```

Comment:
Just wondering if this switch from projects to namespaces was intentional. I'd guess that it should stay as projects.

Reply:
I changed this in response to a previous comment, because I misinterpreted it as a requirement to use 'namespace' instead of 'project' in the docs (up until today I thought they were synonymous). I have reverted back to 'project' for this particular entry.